webui/react/src/components/UPlot/UPlotScatter.tsx

```diff
@@ -4,8 +4,7 @@ import uPlot from 'uplot';
 import UPlotChart, { Options } from 'components/UPlot/UPlotChart';
 import QuadTree, { pointWithin } from 'components/UPlot/UPlotScatter/quadtree';
 import { Range } from 'types';
-
-import { Scale } from '../../types';
+import { Scale } from 'types';
```

Comment:
can we merge imports like this into a single line?

Reply:
found and updated 3 files -- added a ticket to catch other files with a no-duplicate-imports rule